packages/gatsby-transformer-video/README.md

```diff
@@ -0,0 +1,86 @@
+# gatsby-transformer-video
+
+Convert videos into well supported formats via FFMPEG. Supports generating previews as well.
+
+:warning: Converting videos might take a lot of time. Make sure to have a effective caching mechanism in place. See [caching](#caching)
+
+## Features
+
+- Supports file nodes & Contentful assets
+- Defaults optimized for small files with high visual quality
+- Convert videos to h264 and h265
+- Create animated previews as mp4, animated webp and animated gif.
+- Add watermarks to you videos
+- More?
+
+## Installation
+
+```sh
+npm i gatsby-transformer-video
+```
+
```

Comment:
The usage of `fluent-ffmpeg` requires that `ffmpeg` be installed on your machine. Should there be a note regarding this installing?

Reply:
Yes. We might even notify the user on CLI when installing if ffmpeg is missing?

Reply:
If binaries are missing, the CLI outputs: `Cannot find ffprobe`

Writing on an updated README at this very second as well

Reply:
Mentioned it in the readme, the CLI gives an reasonable output by default as well